pages/resources/message.mdx

```diff
@@ -1193,6 +1194,29 @@ Conversation summaries are short, LLM-generated descriptions of a channel's acti
 }
 ```
 
+### Potion Object
```

Comment:
Please move this above the summary object, add an example, and add it to `components/Navigation.tsx`

Reply:
Fixed.